Dockerfile

```diff
@@ -1,20 +1,13 @@
 FROM ubuntu:16.04
 
-LABEL maintainer="zyxep"
+RUN apt-get update
```

Comment:
Splitting the apt-get related commands to separate RUN stages will inflate the image size – completely unnecessarily.

Reply:
Not sure if it does but my main intention was to also fix the docker container file as it currently have an older toolbox when I pull the latest tag

```
~/D/docker (docker-fixes|✔) $ docker pull vapor/toolbox:latest
latest: Pulling from vapor/toolbox
Digest: sha256:c8edddc3b6ffcbbe113790939206e941f40d585993ba15deb1789a2a66ed97a1
Status: Image is up to date for vapor/toolbox:latest
~/D/docker (docker-fixes|✔) $ docker run --rm -it --entrypoint bash vapor/toolbox:latest
root@82c0a78e279e:/vapor# vapor version
Vapor Toolbox: 2.0.4
root@82c0a78e279e:/vapor# swift --version
Swift version 4.0 (swift-4.0-RELEASE)
Target: x86_64-unknown-linux-gnu
root@82c0a78e279e:/vapor#
```

Reply:
I have published my fork of this container and has the updated vapor toolbox https://hub.docker.com/r/encoreptl/vapor-toolbox/

Reply:
Ah, that is because the image build is triggered when there is a new push on github.
So I can’t automatic trigger an update when there is a new tag in the toolbox, let me trigger an update now :)

Reply:
Just did a check for container size and it increased only by 3MB which is negligible considering the caching happening at each RUN
<img width="988" alt="screen shot 2017-10-08 at 7 46 01 pm" src="https://user-images.githubusercontent.com/498669/31322202-94fa6866-ac61-11e7-8b42-e394eb80aab4.png">
<img width="958" alt="screen shot 2017-10-08 at 7 46 15 pm" src="https://user-images.githubusercontent.com/498669/31322203-9840bbb0-ac61-11e7-9e01-e39520b07bae.png">